src/components/Backdrop/Backdrop.scss

```diff
@@ -9,12 +9,16 @@ $backdrop-color: rgba(33, 43, 54, 0.4);
   bottom: 0;
   left: 0;
   display: block;
-  background: $backdrop-color;
+  background-color: $backdrop-color;
```

Comment:
The current version has the issue where the sheet is difficult to see in Windows High Contrast, since it relies on a background color and drop shadow to distinguish it from the content "underneath" (#1213). I'd like to address this before we ship so it doesn't get lost.

The border color doesn't matter, since it should translate to whatever the user's default is.

```
@media screen and (-ms-high-contrast: active) {
    border-color-left: 1px solid transparent;
  }
```

Reply:
👍 Great, thanks for the reminder on this. Will get that implemented!